File: php/1A.php
<?php
fscanf(STDIN, "%d %d %d", $n, $m, $a);

$tilesN = intdiv($n + $a - 1, $a);
$tilesM = intdiv($m + $a - 1, $a);

echo $tilesN * $tilesM;

File: go/1A.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	in := bufio.NewReader(os.Stdin)
	var n, m, a int64
	fmt.Fscan(in, &n, &m, &a)

	tilesN := (n + a - 1) / a
	tilesM := (m + a - 1) / a

	fmt.Println(tilesN * tilesM)
}
